rllib/utils/metrics/tests/test_metrics_logger.py

```diff
@@ -405,20 +385,17 @@ def test_lifetime_stats_behavior():
 
 
 def test_hierarchical_metrics_system():
-    """Test a complete hierarchical metrics system with diverse feature combinations.
-    The test mimics how we use MetricsLogger in EnvRunners, AggregatorActors and the driver (Algorithm object).
+    """Test a hierarchical system of MetricsLoggers."""
```

Comment:
Removing most of this test to test in test_stats instead.
The new test_merging_multiples_rounds test is more comprehensive in testing the expected outcomes of logging values for different types of stats.